public/views/modals/tx-details.html

```diff
@@ -46,7 +37,27 @@ <h4 class="title m0" translate>Details</h4>
         <span class="enable_text_select"> {{btx.labelTo || btx.addressTo}}</span>
       </span>
     </span>
+    </li> 
+
+    <li ng-show="btx.hasMultiplesOutputs" class="line-b p10 oh" 
```

Comment:
when i click on the arrow, it does not expand the list of recipients. perhaps the list is not populated?